src/components/Photo/Selfie.js

```diff
@@ -11,29 +11,92 @@ import style from './style.css'
 
 type State = {
   hasBecomeInactive: boolean,
+  snapshotBuffer: Array<{
+    blob: Blob,
+    base64: string
+  }>,
 }
 
 type Props = {
   onCapture: Function,
   renderFallback: Function,
   trackScreen: Function,
-  inactiveError: Object
+  inactiveError: Object,
+  useMultipleSelfieCapture: boolean,
+  snapshotInterval: number,
 }
 
 export default class Selfie extends Component<Props, State> {
   webcam = null
+  snapshotIntervalRef: ?IntervalID = null
 
   state: State = {
     hasBecomeInactive: false,
+    snapshotBuffer: [],
   }
 
   handleTimeout = () => this.setState({ hasBecomeInactive: true })
 
-  onScreenshotCapture = (blob: Blob, base64: string, sdkMetadata: Object) => {
-    this.props.onCapture({ blob, base64, sdkMetadata })
+  buildCapture = (blob: Blob, base64: string, sdkMetadata: Object, name: string) => ({
+    blob: new File([blob], `${name}.png`, {
```

Comment:
we should get the mimetype from the blob, for both the file extension and the FIle mimetype

Reply:
Done